wagtailnhsukfrontend/templates/wagtailnhsukfrontend/action_link.html

```diff
@@ -1,5 +1,9 @@
 <div class="nhsuk-action-link">
-  <a class="nhsuk-action-link__link" href="{{ external_url }}" {% if new_window %}target="_blank" {% endif %}>
+  {% if internal_page %}
+  <a class="nhsuk-action-link__link internal-page" href="{{ internal_page.url }}">
```

Comment:
`internal-page` and `external-url` classes here don't do anything except make the tests easier. Can we find a way to write the tests so that we don't have to add extra classes to our templates?